WordPress/UITestsFoundation/Screens/ReaderScreen.swift

```diff
@@ -20,6 +20,50 @@ public class ReaderScreen: ScreenObject {
         )
     }
 
+    public func openTheLastPostInApp() {
```

Comment:
Yeah, the idea was really to differentiate it. I was actually a bit uncomfortable with "InApp" because the "InSafari" is somewhat still _in app_. =P

I'll get rid of the articles. =)